example/tbdex_example.dart

```diff
@@ -1,7 +1,13 @@
+import 'dart:convert';
+
 import 'package:tbdex/tbdex.dart';
 
 void main() async {
   final keyManager = InMemoryKeyManager();
+
   final did = await DidJwk.create(keyManager: keyManager);
   print(did.uri);
+
+  final didResolutionResult = DidJwk.resolve(did.uri);
+  print(jsonEncode(didResolutionResult));
```

Comment:
We could add this to the test file if you wanna

Reply:
done! added a few more basic tests